src/client/app/translations/data.json

```diff
@@ -172,6 +172,7 @@
 		"name": "Name",
 		"navigation": "Navigation",
 		"need.more.points": "Need more points",
+		"notify.calibration.needed": "Calibration needed before display", 
```

Comment:
Lets rename this to map.notify.calibration.needed and move it into the map section. Need to do for all 3 languages.

Reply:
I think src/client/app/components/maps/MapViewComponent.tsx needs to be updated with the new name so the translation works properly.